homework-3/Logger.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.IO;
+
+namespace LoggingLibrary
+{
+    public class Logger : IDisposable
+    {
+        private static Logger? s_instance;
```

Comment:
Name of logger instance should be _logger or something more resonable. Question reads as "Include a private static field instance of type Logger" which means to create an instance of Logger not to name it as instance.